### DISCUSSION TITLE
Synology 720+ with 2 nvme and 2 sata. Hibernation issue

### DISCUSSION THREAD

**Discussion by marcomauri78** [007revad/Synology_HDD_db#196] [Q&A]
I've installed two nvme raid 1 using synology_hdd_db script to create a new volume (Samsung 970 evo) I moved container manager on them and i have home assistant running. My os Is 7.2.1-69057 update 3

With this config the hibern of two sata hdd never happened If i switch off the home assistant the hib works with at least three wake up per day

I've added the two nvme for home assistant tò allow the hib of the two sata hdd but in some way the home ass running on the two nvme don't allow the hib

I see this patch but i'm not sure that it works on my os versione

https://www.reddit.com/r/synology/comments/129lzjg/fixing_hdd_hibernation_when_you_have_docker_on/?rdt=47513

**Comment by 007revad** [comment]
Do NOT run that script on DSM 7.2.1

There is a newer version that supports DSM 7.2 here: https://github.com/AlexFromChaos/synology_hibernation_fixer

It incorporates the changes from his reddit posts [here](https://www.reddit.com/r/synology/comments/10cpbqd/making_disk_hibernation_work_on_synology_dsm_7/) and [here](https://www.reddit.com/r/synology/comments/129lzjg/fixing_hdd_hibernation_when_you_have_docker_on/) and a lot of other changes to work in DSM 7, 7.1 and 7.2

**Comment by marcomauri78** [comment]
Hi @007revad 
i tried the above script but with 7.2.1-69057 update 3 and home assistant running on docker the HDD don' hibernate
Thanks
